### PR TITLE
Topics menu

### PR DESCRIPTION
**Pull request by taulinger**
Implementation of a menu panel that displays topics and their catalog

### CODE REVIEW

File: src/modules/commons/components/checkbox/Checkbox.js [hunk 1]
```diff
@@ -63,6 +63,10 @@ export class Checkbox extends BaElement {
 			this._onToggle(event);
 		};
 
+
+		// const root = document.querySelector('ba-checkbox').shadowRoot;
+		// root.addEventListener('click', alert('click'));
```
Comment (thiloSchlemmer) [inline src/modules/commons/components/checkbox/Checkbox.js:68]: this commented out code needs an additional comment, otherwise it should be removed

Comment (taulinger) [reply src/modules/commons/components/checkbox/Checkbox.js]: thx, done

File: src/modules/topics/components/menu/catalog/CatalogNode.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		this.render();
+	}
+
+
+	initialize() {
+		this._level = this.getAttribute('level') ? parseInt(this.getAttribute('level')) : 0;
+	}
+
+	onStateChanged() {
+		//nothing to to here: initial rendering does it
```
Comment (thiloSchlemmer) [inline src/modules/topics/components/menu/catalog/CatalogNode.js:34]: Typo

Comment (taulinger) [reply src/modules/topics/components/menu/catalog/CatalogNode.js:34]: thx

**Review by thiloSchlemmer — APPROVED** [review]

**Comment by thiloSchlemmer** [comment]
Comments inline

**Comment by thiloSchlemmer** [comment]
sorry for recognizing this so late but....

1. the `ba-dev-info`-component seems to be sticked with the position of topic-nodes/leafs and moves up/down when the topics are scrolled up/down

2. the i-Symbol in the information-Button is partially hidden.

**Comment by taulinger** [comment]
> sorry for recognizing this so late but....
> 
>     1. the `ba-dev-info`-component seems to be sticked with the position of topic-nodes/leafs and moves up/down when the topics are scrolled up/down
> 
>     2. the i-Symbol in the information-Button is partially hidden.

Created an issue for that.

**Pull request merged**